reference-architecture/aws-ansible/playbooks/openshift-setup.yaml

```diff
@@ -0,0 +1,34 @@
+- include: /usr/share/ansible/openshift-ansible/playbooks/byo/config.yml 
+  vars:
+    debug_level: 2
+    openshift_debug_level: "{{ debug_level }}"
+    openshift_node_debug_level: "{{ node_debug_level | default(debug_level, true) }}"
+    osm_controller_args:
+      cloud-provider:
+      - "aws"
+    osm_api_server_args:
+      cloud-provider:
+      - "aws"
+    openshift_node_kubelet_args:
+      max-pods:
+      - "100"
```

Comment:
I'm not sure you want to override the max-pods value. We did for the demo environments, but we also knew what the workload running was going to be.